week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -8,12 +8,35 @@
   Create a function which checks if the bush has ALL PINK berries and is safe for the astronauts to eat from the bush.
 
   Use the tests to confirm which message to return
-*/
 
-function bushChecker() {
+  This exercise can be solved in a few different ways. One way might include the array methods
+  .some() and .every(). The .some() method tests to see if some of the values (at least 1) in an array 
+  match what you're looking for and returns true or false. .every() will only return true 
+  if all values match watch you're looking for. Let's first look at an example that will 
+  teach you how to use these methods.
+ */
+
+let array = [12, 73, 92, 45, 100, 14, 61];
+
+array.some((value) => {return (value % 2 == 0)}); /* this will return true as SOME values 
+will have a remainder of 0 i.e. they are even numbers*/
+
+array.every((value) => {return (value % 2 == 0)}); /* this will return false as not ALL 
+values will have a remainder of 0 i.e. there are some odd numbers in the array too*/
+
+/* + + + + + + + + + + + + + + */
+
+/* Now try to complete the exercise */
+
+function bushChecker(arr) {
+if(arr.every(value=>value==='pink'))
```

Comment:
```suggestion
if(arr.every(value => value === 'pink'))
```
Can you think of another variable name besides `value` - think about what each item in the array represents.